pkg/components/azure-arc/image/script.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Type of action: register or remove the Lokomotive cluster from Azure Arc.
+action=$1
+
+ca="$(cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt | base64 | tr -d "\n")"
+namespace="$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)"
+token="$(cat /var/run/secrets/kubernetes.io/serviceaccount/token | tr -d "\n")"
+server="https://kubernetes.default.svc"
+
+echo "
+apiVersion: v1
+kind: Config
+clusters:
+- name: default-cluster
+  cluster:
+    certificate-authority-data: ${ca}
+    server: ${server}
+contexts:
+- name: default-context
+  context:
+    cluster: default-cluster
+    namespace: ${namespace}
+    user: default-user
+current-context: default-context
+users:
+- name: default-user
+  user:
+    token: ${token}
+" > sa.kubeconfig
```

Comment:
Maybe we can put this part in function as well, so we don't mix function declaration with executed code?